proto/osmosis/concentrated-liquidity/genesis.proto

```diff
@@ -68,6 +68,9 @@ message GenesisState {
 
   uint64 next_position_id = 4
       [ (gogoproto.moretags) = "yaml:\"next_position_id\"" ];
+
+  uint64 next_incentive_record_id = 5
```

Comment:
@p0mvn one quick thought;
instead of creating new incentive at each epoch for numEpochFilled > 1, why dont we just sum up the coins. With this we create a new incentive record for the same coins too 

for ex: if we have 2 incRecord
id: 1 
coin: 100atom
minuptime: 1ns
... 

incRecord 
id: 2
coin: 100atom
minuptime: 1ns
... 


instead why dont we just do 
coin: 200atom
minuptime: 1ns

Reply:
Because the emission rates might differ?

There is one optimization that I'm thinking about.

Currently, we create an incentive record for every coin in gauge.

Instead, we could have one incentive record for all coins in gauge if we were to introduce multiple emission rates per incentive record.

I will make an issue to track this. Don't think this is launch blocking though

Reply:
https://github.com/osmosis-labs/osmosis/issues/5531